src/OrchardCore/OrchardCore.AsModule/OrchardCore.AsModule.props

```diff
@@ -8,6 +8,7 @@
   <PropertyGroup>
     <ModuleManifest>Name: $(MSBuildProjectName)</ModuleManifest>
     <ExcludedFiles>**\*.cs;*.csproj*;obj\**;bin\**;Assets.json;Assets\**;Properties\**;*.props;*.targets;.vs\**</ExcludedFiles>
+    <Temp Condition="'$(Temp)' == ''">..\..\..\temp</Temp>
   </PropertyGroup>
```

Comment:
Yes, probably you just need add something in the document to say additional argument is required when you build the application with command line in osx/linux:

`
dotnet msbuild /t:build /p:Temp:xxxx
`